ui/app/templates/components/job-page/periodic.hbs

```diff
@@ -25,7 +25,7 @@
           <span class="term">
             Cron
           </span>
-          {{@job.periodicDetails.Spec}}
+          {{#each @job.periodicDetails.Specs as |spec|}}{{spec}}; {{else}}{{@job.periodicDetails.Spec}}{{/each}}
```

Comment:
```suggestion
          {{#each @job.periodicDetails.Specs as |spec|}}
            <span class="bumper-right tag">{{spec}}</span>
          {{else}}
            <span class="tag">{{@job.periodicDetails.Spec}}</span>
          {{/each}}
```

While I think `;` is not a reserved character for cron expression, it does look quite a bit like a `,` which is valid, so it may be tricky to notice the separation between expression:

<img width="738" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/2e28f5bd-f396-41da-b31d-b4b53686c300">

I think using a pill here makes the distinction easier to read:
<img width="781" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/29ee1e98-c082-4e06-b70e-dbc160e8c0b7">


But pinging @philrenaud and @juliezzhou for extra input 😄

Reply:
Pills look great! I am not sure if `cron` is singular where `crons` is plural, but if that is the case, you can also switch your `term` code to be something like: `{{pluralize "Cron" @job.periodicDetails.Specs.length}}`

Reply:
Thank you for this suggestion! Frontend is very hard for me :smile:

Reply:
> Pills look great! I am not sure if `cron` is singular where `crons` is plural, but if that is the case, you can also switch your `term` code to be something like: `{{pluralize "Cron" @job.periodicDetails.Specs.length}}`

Ah nice, I think in this case we may need to handle zero length, which indicates that `cron` is being used.

```diff
          <span class="term">
-           Cron
+           {{pluralize "Cron" (or @job.periodicDetails.Specs.length 1)}}
          </span>
```

Reply:
Thanks, I lost it